pipestat/reports.py

```diff
@@ -40,21 +41,30 @@
 class HTMLReportBuilder(object):
     """Generate HTML summary report for project/samples"""
 
-    def __init__(self, prj):
+    def __init__(self, prj, portable):
```

Comment:
I think you want to add default value here

Reply:
I had it set below, but I moved it to the parameter level as you suggested.